pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -261,3 +264,25 @@ func createTopologyAttributes(policy string, scope string) v1alpha2.AttributeLis
 		},
 	}
 }
+
+func updateAttribute(attrList *v1alpha2.AttributeList, attrInfo v1alpha2.AttributeInfo) {
+	if nil == attrList {
```

Comment:
```suggestion
	if attrList == nil {
```
I know it is correct but it looks odd to have nil first.